generated/nidcpower/nidcpower/lcr_load_compensation_spot.py

```diff
@@ -0,0 +1,144 @@
+import ctypes
+import numbers
+import platform
+
+import nidcpower._visatype
+import nidcpower.enums as enums
+
+
+# This class is an internal implementation detail
+# ctypes definition
+# Name must match exactly what the name of the structure type is named in the C API.
+class struct_NILCRLoadCompensationSpot(ctypes.Structure):  # noqa N801
+    if platform.system() == "Windows" and platform.architecture()[0] == "64bit":
+        _pack_ = 8
+    elif platform.system() == "Linux" or platform.system() == "Darwin":
+        _pack_ = 4
+    else:
```

Comment:
This code for determining packing is really following the exact logic in the public "nidcpower.h" header file:
```
#if defined(_WIN64)
#pragma pack(push,8)
#elif defined(__linux__) || defined(__APPLE__)
#pragma pack(push,4)
#else
#pragma pack(push,1)
#endif
```
I don't think we should add explicit constraints here that would make it inconsistent with the public header file.

> Also, this code for determining packing is probably duplicated with other similar structs. Single source it please. Consider doing it in its own PR.

There is currently only one instance of usage of `_pack_` in the entire nimi-python, i.e. in [`niscope.struct_niScope_wfmInfo`](https://github.com/ni/nimi-python/blob/master/src/niscope/custom_types/waveform_info.py), and it is using a different logic (i.e. its packing is fixed at 8 bytes, same as what have been defined in "niscope.h" public header file).

I have created #1777 for single-sourcing such code, but I don't think it should be implemented now because:
- There isn't any driver using this at the moment
- Different drivers may require different logic for their structs